tcms/core/tests/test_ajax.py

```diff
@@ -91,3 +97,89 @@ def test_update_default_tester_non_existing_user(self):
         self.assertEqual(result['response'], 'Default tester not found!')
 
         self._assert_default_tester_is(None)
+
+
+class Test_Tag(BasePlanCase):
```

Comment:
inherit from test.TestCase b/c you'e not using any functionality provided by the BasePlanCase class.